Assets/MRTK/Tools/BuildWindow/Arm64Workaround.cs

```diff
@@ -34,65 +34,74 @@ public void OnPostprocessBuild(BuildReport report)
+                        {
+                            return;
+                        }
 
-                    buildCommandLineText = buildCommandLine.Value;
-                    rebuildCommandLineText = rebuildCommandLine.Value;
+                        buildCommandLineText = buildCommandLine.Value;
+                        rebuildCommandLineText = rebuildCommandLine.Value;
+                    }
                 }
-            }
 
-            // This is unexpected for the workaround but if this happens, we should abort.
-            if (buildCommandLineText.Length == 0)
-            {
-                return;
-            }
+                // This is unexpected for the workaround but if this happens, we should abort.
+                if (buildCommandLineText.Length == 0)
+                {
+                    return;
+                }
 
-            buildCommandLineText += WorkaroundCompilerFlags;
+                buildCommandLineText += WorkaroundCompilerFlags;
 
-            // The newly added node has to be appended to the end of the Project node, otherwise
-            // its values will not take effect (i.e. the last setter of the given values will win)
-            XElement newPropertyGroup = new XElement(root.GetDefaultNamespace() + "PropertyGroup");
-            newPropertyGroup.SetAttributeValue("Condition", Arm64Condition);
+                // The newly added node has to be appended to the end of the Project node, otherwise
+                // its values will not take effect (i.e. the last setter of the given values will win)
+                XElement newPropertyGroup = new XElement(root.GetDefaultNamespace() + "PropertyGroup");
+                newPropertyGroup.SetAttributeValue("Condition", Arm64Condition);
 
-            XElement newBuildCommandLine = new XElement(root.GetDefaultNamespace() + "NMakeBuildCommandLine");
-            XElement newRebuildCommandLine = new XElement(root.GetDefaultNamespace() + "NMakeReBuildCommandLine");
-            newBuildCommandLine.SetValue(buildCommandLineText);
-            newRebuildCommandLine.SetValue(rebuildCommandLineText);
+                XElement newBuildCommandLine = new XElement(root.GetDefaultNamespace() + "NMakeBuildCommandLine");
+                XElement newRebuildCommandLine = new XElement(root.GetDefaultNamespace() + "NMakeReBuildCommandLine");
+                newBuildCommandLine.SetValue(buildCommandLineText);
+                newRebuildCommandLine.SetValue(rebuildCommandLineText);
 
-            newPropertyGroup.Add(newBuildCommandLine);
-            newPropertyGroup.Add(newRebuildCommandLine);
+                newPropertyGroup.Add(newBuildCommandLine);
+                newPropertyGroup.Add(newRebuildCommandLine);
 
-            root.Add(newPropertyGroup);
-            root.Save(path);
+                root.Add(newPropertyGroup);
+                root.Save(path);
+            }
+            catch (Exception) 
```

Comment:
Do we want to log to let the user know we tried and failed? "Hey heads up if you're building ARM64, we tried to work around a known issue but failed. See [issue] for more info."